### DISCUSSION TITLE
Change the height of the row

### DISCUSSION THREAD

**Discussion by abenedet78** [jcwillox/lovelace-paper-buttons-row#112] [Q&A]
Thank you for developing this. It's awesome and exactly what an individual with limited coding skills needs. 
One question: is there a way to change the height of the row?

**Comment by LeiChat** [comment]
Being able to configure card/row padding would be a feature request from me too.

In the meantime, I'm using this styling with [card_mod](https://github.com/thomasloven/lovelace-card-mod) to reduce the padding:
```
card_mod:
  style: |
    .card-header {
      font-size: 18px !important;
      padding-bottom: 2px;
      padding-top: 0px;
    }
    .card-content {
      padding-bottom: 1px;
      padding-left: 0;
      padding-right: 0;
    }
```

**Reply by LeiChat** [reply]
@abenedet78 Hi. I was browsing the discussions and found [a reply that mentions a way of amending the height](https://github.com/jcwillox/lovelace-paper-buttons-row/discussions/71#discussioncomment-2303981).
